app/src/main/java/ar/com/wolox/android/cookbook/googleLogin/model/UserGoogle.kt

```diff
@@ -0,0 +1,9 @@
+package ar.com.wolox.android.cookbook.googleLogin.model
+
+import com.google.android.gms.auth.api.signin.GoogleSignInAccount
+
+data class UserGoogle(private val user: GoogleSignInAccount) {
```

Comment:
This name sounds off. Can't we just use `GoogleAccount`?

Reply:
Yes, that's better. In the original project this name made more sense. :upside_down_face: